eng/pipelines/templates/jobs/apiview-review-gen-python.yml

```diff
@@ -0,0 +1,44 @@
+pr: none
+
+trigger: none
+
+pool:
+  name: azsdk-pool-mms-ubuntu-2004-general
+  vmImage: MMSUbuntu20.04
+
+parameters:
+  - name: Reviews
+    type: string
+    default: ''  
+  - name: APIViewURL
+    type: string
+    default: '' 
+  - name: StorageContainerUrl
+    type: string
+    default: ''
+
+variables:
+  PythonIndexUrl: 'https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-python/pypi/simple/'
+  PythonVersion: '3.10.2'
```

Comment:
Do we need to have this specific version, or should we accept the default version in use-python-version.yml? It doesn't look like there is a default version currently but we probably should consider that.

Reply:
Python team wanted to pin API reviews to a specific version after a python version upgrade caused issues for reviews when upgraded to 3.10.1.